src/domain/actuator_server_app.py

```diff
@@ -5,13 +5,16 @@
 
 from fastapi import FastAPI, HTTPException
 from fastapi.responses import JSONResponse
+from starlette.middleware.cors import CORSMiddleware
```

Comment:
I don't remember, is starlette another package we need to install or is included in `fastapi`?

Reply:
It should be included in `fastapi `because I didn't install it explicitly.